source/wp-content/themes/wporg-showcase-2022/src/site-screenshot/style.scss

```diff
@@ -1,22 +1,24 @@
 .wp-block-wporg-site-screenshot {
+	overflow: hidden;
+	--local--focus--border-color: var(--wp--custom--link--color--text);
+
+	&:focus-within {
+		outline: 1.5px solid var(--local--focus--border-color);
+		outline-offset: -1.5px;
+	}
+
+	a {
+		display: block;
+
+		&:focus,
+		&:focus-visible {
+			outline: none;
+		}
+	}
+
 	img {
 		width: 100%;
 		vertical-align: middle;
 		aspect-ratio: 16 / 9;
 	}
 }
-
-// If we have a parent block with a border radius, we want to clip the contents.
-.wp-block-group[style*="border-radius"]:has(.wp-block-wporg-site-screenshot) {
```

Comment:
These rules were to allow for editor customization of the border-radius. We may want to keep them.

Reply:
We don't need it now that the block itself supports the border radius.